Marlin/language_de.h

```diff
@@ -90,29 +90,29 @@
 #define MSG_PID_I                           "PID I"
 #define MSG_PID_D                           "PID D"
 #define MSG_PID_C                           "PID C"
-#define MSG_ACC                             "A"
-#define MSG_VXY_JERK                        "V xy Ruck"
-#define MSG_VZ_JERK                         "V z  Ruck"
-#define MSG_VE_JERK                         "V e  Ruck"
-#define MSG_VMAX                            "V max " // space by purpose
+#define MSG_ACC                             "Beschleunig."
+#define MSG_VXY_JERK                        "V xy Bew-Mom."
+#define MSG_VZ_JERK                         "V z  Bew-Mom."
+#define MSG_VE_JERK                         "V e  Bew-Mom."
+#define MSG_VMAX                            "V Max " // space by purpose
 #define MSG_X                               "X"
 #define MSG_Y                               "Y"
 #define MSG_Z                               "Z"
 #define MSG_E                               "E"
-#define MSG_VMIN                            "V min"
-#define MSG_VTRAV_MIN                       "VTrav min"
-#define MSG_AMAX                            "A max " // space by purpose
-#define MSG_A_RETRACT                       "A Rückzug"
-#define MSG_A_TRAVEL                        "A Reise"
-#define MSG_XSTEPS                          "X steps/mm"
-#define MSG_YSTEPS                          "Y steps/mm"
-#define MSG_ZSTEPS                          "Z steps/mm"
-#define MSG_ESTEPS                          "E steps/mm"
+#define MSG_VMIN                            "V Min"
+#define MSG_VTRAV_MIN                       "V Beweg. Min"
+#define MSG_AMAX                            "A Max " // space by purpose
```

Comment:
Dann vielleicht "Urlaub"? Just kidding... ;-)
Eigentlich ist der zurückgelegte Weg gemeint - wie drücken wir den in 17 Zeichen aus?